etna/transforms/math/limit.py

```diff
@@ -0,0 +1,122 @@
+from typing import List
+import numpy as np
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.datasets import set_columns_wide
+from etna.transforms.base import ReversibleTransform
+from etna.transforms.utils import match_target_quantiles
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders"""
+
+    def __init__(self, in_column: str, lower_bound: float = -1e10, upper_bound: float = 1e10):
+        """
+        Init LimitTransform.
+
+        Parameters
+        ----------
+        in_column:
+            column to apply transform
+        lower_bound:
+            lower bound for the value of the column
+        upper_bound:
+            upper bound for the value of the column
+        """
```

Comment:
You should also add "Raises" section [see](https://github.com/etna-team/etna/blob/1d20ffc07f90e76cc9fb0717a6e9887551ebd442/etna/pipeline/base.py#L525)